### PR TITLE
fix partial arg match - close issue #125

### PR DESCRIPTION
**Pull request by lgatto**

### CODE REVIEW

File: R/rbindFill.R [hunk 1]
```diff
@@ -58,7 +58,7 @@ rbindFill <- function(...) {
     allnms <- unique(names(allcl))
     allcl <- allcl[allnms]
 
-    for (i in seq(along=l)) {
+    for (i in seq(along.with = l)) {
```
Comment (sgibb) [inline R/rbindFill.R:61]: We use `seq_along` everywhere else.

```suggestion
    for (i in seq_along(l)) {
```

**Review by sgibb — APPROVED** [review]
Nice catch.

**Comment by lgatto** [comment]
These warnings start to pop up since I set the following in my `.Profile`

```
options(warnPartialMatchArgs = TRUE,
        warnPartialMatchDollar = TRUE,
        warnPartialMatchAttr = TRUE)
```

**Review by jorainer — APPROVED** [review]
Thanks! feel free to merge and push!

**Comment by lgatto** [comment]
Pushed to Bioc.

**Pull request merged**